.github/scripts/generate_changelog.py

```
@@ -79,17 +79,29 @@
-        if m["title"] == title:
-            return m["number"]
+    page = 1
+    recent_titles = []
+    while True:
+        params = {
+            "state": "all",
+            "per_page": 100,
+            "page": page,
+            "direction": "desc",  # most recent first so we find it quickly
+        }
+        resp = requests.get(url, headers=HEADERS, params=params)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)